src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCost = 40;
+
+  if (days > 6) {
+    return carCost * days - 50;
+  }
+
+  if (days > 2) {
+    return carCost * days - 20;
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item. You should create variables for main values - 20, 50, they shouldn't be just magic numbers